src/app/shared/services/vm-snapshot.service.ts

```diff
@@ -1,17 +1,22 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
+import * as omitBy from 'lodash/omitBy';
 import * as pickBy from 'lodash/pickBy';
 import { Observable, of } from 'rxjs';
 import { map, switchMap } from 'rxjs/operators';
+import { VirtualMachine } from '../../vm';
+import { virtualMachineEntityName } from '../../vm/shared/vm.service';
 import { BackendResource } from '../decorators';
 import { SnapshotFromVmSnapshotParams } from '../interfaces';
 import { VmSnapshot } from '../models/vm-snapshot.model';
 import { AsyncJobService } from './async-job.service';
 import { BaseBackendService, CSCommands } from './base-backend.service';
 
+export const vmSnapshotEntityName = 'VMSnapshot';
+
 @Injectable()
 @BackendResource({
-  entity: 'VMSnapshot',
+  entity: vmSnapshotEntityName,
```

Comment:
I guess constant usage is redundant here (except the situations when `vmSnapshotEntityName` reused somewhere).